tracing-appender/src/rolling.rs

```diff
@@ -42,6 +85,10 @@ impl io::Write for RollingFileAppender {
 /// The appender returned by `rolling::hourly` can be used with `non_blocking` to create
 /// a non-blocking, hourly file appender.
 ///
+/// The location of the log file will be specified the `directory` passed in.
+/// `file_name_prefix` specifies the prefix of the log file. It is appended by the
+/// current date and hour.
```

Comment:
```suggestion
/// The directory of the log file is specified with the `directory` argument.
/// `file_name_prefix` specifies the _prefix_ of the log file. `RollingFileAppender` 
/// adds the current date and hour to the log file.
```

Note: we should probably show an example of what the fully fleshed-out log file path looks like here.